packages/react/src/index.ts

```diff
@@ -0,0 +1 @@
+export * from '@backtrace/browser';
```

Comment:
I have mixed feelings about that - I know we want to have something that compiles and generates an output, however the first thing that we need to do just after adding anything is to remove this line of code